src/config.ts

```diff
@@ -357,9 +357,13 @@ function normalizeScripts(cwd: string, scripts: RawScripts): BuildScript[] {
       }
       let dirDisk = cmdArr[0];
       const dirUrl = to || `/${cmdArr[0]}`;
+
+      // mount:web_modules is a special case script where the fromDisk
+      // arg is harcoded to match the internal dependency dir
       if (scriptId === 'mount:web_modules') {
```

Comment:
@FredKSchott What do you think about doing this instead?

```js
dirDisk = dirDisk.replace('$WEB_MODULES', dependenciesLoc);
```

That would make the [Script Variables](https://www.snowpack.dev/#script-variables) section of the docs more accurate and technically it'd be a little bit more flexible. Not sure that anyone needs/wants that flexibility though.

Reply:
+1, I think that's a good idea. Feel free to tackle in a future PR